index.js

```diff
@@ -7,40 +7,66 @@ const app = express();
 const autovoice = require('./bin/lib/autovoice');
 const formatContentForReading = require('./bin/lib/formatContentForReading');
 
+const authS3O = require('s3o-middleware');
 
 var requestLogger = function(req, res, next) {
     debug("RECEIVED REQUEST:", req.method, req.url);
     next(); // Passing the request to the next handler in the stack.
 }
 
 app.use(requestLogger);
-app.use('/static', express.static('static'));
 
-app.get('/', (req, res) => {
-	res.sendFile(path.join(__dirname + '/static/index.html'));
-});
+// these routes do *not* have s3o
+
+app.use('/static', express.static('static'));
 
 app.get('/__gtg', (req, res) => {
 	res.status(200).end();
 });
 
+const PODCAST_TOKEN = process.env.PODCAST_TOKEN;
```

Comment:
where it is will cause the app to fail on startup, which hopefully means heroku will not deploy it